wildfly/connectors/infinispan/pom.xml

```diff
@@ -33,7 +33,11 @@
 				<version>${project.version}</version>
 				<type>test-jar</type>
 			</dependency>
-
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-databind</artifactId>
+                <version>2.10.0</version>
```

Comment:
Should we bump everything to 2.10?

Also I know we chatted about this, but wanted to make sure.  The databind dependency is not declared in module, only here.  Should/could it be test scoped?

Reply:
Yes, this is test scope only. Added the scope and moved the version to a property, and updated the commit.